plugins/com.python.pydev.analysis/src/com/python/pydev/analysis/refactoring/quick_fixes/AddTokenAndImportStatement.java

```diff
@@ -198,37 +198,43 @@ public void createTextEdit(ComputedInfo computedInfo) {
                 String strToAdd = ", " + realImportHandleInfo.getImportedStr().get(0);
 
                 String line = PySelection.getLine(document, endLine);
+                line = line.trim();
```

Comment:
I think this `trim()` may break things as it's trimming the line used for other later on in the line and that'd break offsets/ line len... I think a test case where you have some spaces before/after the import will show the breakage (so, please create a test case with `*\t*\tfrom a import c ,*\t*\t` -- consider the `*` to be the whitespaces and `\t` to be tabs).